snarky_monad_lib/dune

```diff
@@ -0,0 +1,9 @@
+(library
+ (name snarky_monad_lib)
```

Comment:
Note that _lib is not necessary, and I feel it is a bit redundant as we know it is a library. API users would have to add this suffix, which might not sound natural. Although, I think it's mostly to follow the naming convention in Snarky you're doing this?

Reply:
Yeah I just followed the naming convention from `tuple_lib`. And in fact you will find your dune file comments apply equally to there because I took that dune file as a template.